src/data/markdown/translated-guides/en/02 Using k6/06 Test life cycle.md

```diff
@@ -26,10 +31,26 @@ export function teardown(data) {
 
 </CodeGroup>
 
+## A quick overview of test stages
+
+This table provides the essential information about each stage.
+The rest of this page goes into deeper technical detail.
+
+| Test stage      | Used to                                                    | Example                                                                                 | Called                                                                             | Required? |
+|-----------------|------------------------------------------------------------|-----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|-----------|
+| **1. init**     | Load local files, import modules, declare global variables | Open JSON file, Import module                                                           | Once per VU\*                                                                      | Optional  |
+| **2. Setup**    | Set up data for processing, share data among VUs           | Call API to start test environment                                                      | Once                                                                               | Optional  |
+| **3. VU code**  | Run the test script                                        | Make https requests, validate responses                                                 | Per VU, for as many iterations and as long a duration as the test options required | Required       |
```

Comment:
```suggestion
| **3. VU code**  | Run the test function, usually the `default` one                                        | Make https requests, validate responses                                                 | Once per iteration, as many times  as the test options required | Required       |
```

I would want to make it clear that we are calling *a function*, usually the `default` one, not a "script", which has a definition in ECMAScript which is more or less the whole file. But in practice "run the test script" is probably more technically correct to be said about the "init code' ;) 

Additionally, while the last part is now more vague it is also more correct as otherwise it doesn't really work for arrival rates or `shared-iterations` , especially as the duration is just a limit in the second one, not how long it should run.